README.md

```diff
@@ -35,6 +35,7 @@ The Kubernetes stack of RADAR-base platform.
 - [Uninstall](#uninstall)
 - [Update charts](#update-charts)
 - [Feedback and Contributions](#feedback-and-contributions)
+- [Service-specific documentation](#service-specific-documentation)
```

Comment:
I think is section should be after [Usage and accessing the applications]

Reply:
Fixed!